README.md

```diff
@@ -19,7 +19,7 @@ Currently this looks to test whether the required basic behaviour is possible. I
 
 ## Aims
 
-Users should be able to deploy the bridge into their cluster with a provider selected like "AzureRMv1.2". It should then:
+Users should be able to deploy the bridge into their cluster with a provider selected like `AzureRMv1.2`. It should then:
 
 1. Generation of mappings TF->CRD should take place at startup/in cluster for ease of use. (Build time step considered but discarded due to added complexity on the user)
```

Comment:
Change to improve flow:
Generation -> Generate

Reply:
Agree, done.